posthog-node/src/edge/runtime/index.ts

```diff
@@ -0,0 +1,5 @@
+import { RuntimeTypes } from 'posthog-node/src/types'
+
+export const runtime = {
+  stackParser: () => [],
+} as RuntimeTypes
```

Comment:
Runtime specific implementations live inside of the `runtime` directory for each package